cla_backend/libs/eligibility_calculator/calculator.py

```diff
@@ -331,17 +334,21 @@ def is_disposable_capital_eligible(self):
         return self.disposable_capital_assets <= limit
 
     def is_eligible(self):
-        cfe_request_dict = self._translate_case()
-
-        # this line fails the lint check as we're ignoring the response
-        # cfe_response = requests.post(settings.CFE_URL, json = cfe_request_dict)
-        requests.post(settings.CFE_URL, json=cfe_request_dict)
+        self._do_cfe_civil_check()
 
         return self._legacy_check()
 
+    def _do_cfe_civil_check(self):
+        cfe_request_dict = self._translate_case()
+
+        cfe_civil_response = requests.post(settings.CFE_URL, json=cfe_request_dict)
+        cfe_data = json.loads(cfe_civil_response.content)
```

Comment:
This is more natural:
```suggestion
        cfe_data = cfe_civil_response.json()
```